### PR TITLE
Prodsetting: flere statsborgerskap og PDL auth

### PR DESCRIPTION
**Pull request by stianStensli**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -2,87 +2,121 @@ name: Build, push and deploy
...
+        if: ${{ steps.card.outputs.success }}
+        with:
+          key: ${{ secrets.TRELLO_KEY }}
+          token: ${{ secrets.TRELLO_TOKEN }}
+          card-id: ${{ steps.card.outputs.card-id }}
+          attachment-type: branch
   test:
     name: Run tests
     runs-on: ubuntu-latest
-    if: github.ref != 'refs/heads/dev' && github.ref != 'refs/heads/master'
```
Comment (stianStensli) [inline .github/workflows/main.yaml:11]: No test in dev? 
i think I disagree

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -2,87 +2,121 @@ name: Build, push and deploy
...
+        uses: actions/checkout@v3
       - name: Deploy application
         uses: nais/deploy/actions/deploy@v1
         env:
           APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
           CLUSTER: dev-fss
           RESOURCE: nais-dev.yaml
-          VAR: version=${{ env.IMAGE_TAG }}
-
+          VAR: version=${{ env.IMAGE_TAG }},namespace=pto
```
Comment (stianStensli) [inline .github/workflows/main.yaml:99]: ,namespace=pto is unnecessary

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -2,87 +2,121 @@ name: Build, push and deploy
...
       - name: Set up cache
-        uses: actions/cache@v1
+        uses: actions/cache@v3
         with:
           path: ~/.m2/repository
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-maven-
       - name: Build maven artifacts
-        run: mvn -B package
+        run: mvn -Dgithub.token=${{ secrets.GITHUB_TOKEN }} -B package -D skipTests
```
Comment (stianStensli) [inline .github/workflows/main.yaml:72]: Is -Dgithub.token=${{ secrets.GITHUB_TOKEN }} necessary?

**Comment by stianStensli** [comment]
Ikke klar. Feiler for bulk request

File: .github/workflows/main.yaml (hunk 1 above)
Comment (dragutin-nav) [reply .github/workflows/main.yaml:11]: Will fix..

Comment (dragutin-nav) [reply .github/workflows/main.yaml:11]: Although its the same in veilarbportefolje..

Comment (dragutin-nav) [reply .github/workflows/main.yaml:11]: Tests will run in dev... condition is `github.ref != 'refs/heads/master'`

File: .github/workflows/main.yaml (hunk 2 above)
Comment (dragutin-nav) [reply .github/workflows/main.yaml:99]: Fixed

File: .github/workflows/main.yaml (hunk 3 above)
Comment (dragutin-nav) [reply .github/workflows/main.yaml:72]: I will try without it

**Pull request closed without merging**